src/lib/services/project-service.ts

```diff
@@ -412,6 +414,7 @@ export default class ProjectService {
         const role = await this.accessService.getRole(roleId);
         const group = await this.groupService.getGroup(groupId);
         const project = await this.getProject(projectId);
+        if (!group.id) return;
```

Comment:
If group is not found should we exit or assign no group?

Reply:
Currently we'd fail with an error. I don't think this situation should ever happen but because some other type in unrelated part of the codebase we have to deal with it here. Another option would be to throw IllegalStateException. I'd expect one of our existing tests to tell me if I broke something.

Reply:
If we'd fail, then I'd fail here (fail fast), but if we return we're changing the behavior. Might not be that bad because it's an edge case, but the takeaway is that we need to be careful

Reply:
After this comment I'll default to replicating current behavior and failing but with a more explicit error message